docs/content/preview/secure/authentication/ldap-authentication-ysql.md

```diff
@@ -87,6 +87,45 @@ The configurations supported for search + bind mode.
 | `ldapsearchfilter` | (empty) | The search filter to use when doing search + bind authentication. |
 | `ldapurl` | (empty) | An RFC 4516 LDAP URL. This is an alternative way to write LDAP options in a more compact and standard form. |
 
+## Create secrets for Kubernetes
+
+In Kubernetes, you can create secrets with sensitive information in the namespace where you are running YugabyteDB. Later, when creating universe pods, you can refer to those secrets in environment variables to use in configuring LDAP.
+
+To create a secret:
+
+1. Kubernetes expects secret data to be in base64 format. Run the following command in a shell to convert a password into base64 format:
+
+    ```sh
+    printf 'PASSWORD' | base64
+    ```
+
+    Replace PASSWORD with your password.
+
+1. Add the following contents to a yaml file:
+
+    ```yaml
+    apiVersion: v1
+    kind: Secret
+    data:
+      <KEY NAME>: <Base64 password>
+    metadata:
+      annotations:
+      name: <SECRET NAME>
+    type: Opaque
+    ```
+
+    Replace `Base64 password` with the base64 password you generated.
+
+    Provide a key name and secret name. These are [what] used to [what]
+
+1. Run the following command to create the secret in the namespace running YugabyteDB:
+
+    ```sh
+    kubectl apply -n <namespace> -f <path-to-yaml-file>
+    ```
+
+    Replace namespace with the namespace running YugabyteDB. Replace path-to-yaml-file with the full path to the yaml file you created in the preceding step.
+
```

Comment:
@govardhanjalla Can you please provide instructions for creating the environment variable that will then be used when using the secret, for example in the override in YBA. Something like:
To use the secret [when creating a cluster? some context], add it to an environment variable. For example:

- steps